src/AutoRest.CSharp.V3/Configuration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+using AutoRest.CSharp.V3.JsonRpc;
+
+namespace AutoRest.CSharp.V3
+{
+    internal class Configuration
+    {
+        public string OutputPath { get; private set; }
+
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
+        private Configuration() { }
```

Comment:
Instead of disabling the warning I would simply pass in the value to the constructor in your Create method below.

Reply:
This file is intended to be a grab-bag of stuff from the configuration that would be useful during code generation. All of the properties will be like this, so doing on the constructor means I can just add properties without worrying about it. However, I would like to be explicit about what is nullable versus non-nullable.

Reply:
Why is adding them to the constructor so much harder then setting them in the block below?